go.mod

```diff
@@ -1,6 +1,6 @@
 module k8s.io/org
 
-go 1.20
+go 1.21
```

Comment:
Let's bump to go1.22 directly?

Reply:
I've updated it to go1.22 as suggested. Please let me know if there's anything else that needs to be reviewed or adjusted.